manifests/interface.pp

```diff
@@ -440,6 +448,18 @@
 
   # Resources
 
+  if $restart_all_nic {
```

Comment:
I would add here a condition like: or kernel != 'Linux' 
so that the per interface command is used only when we are sure it works